docs/stacks-guide-aws.md

```diff
@@ -0,0 +1,896 @@
+  reclaimPolicy: Delete
+  providerRef:
+    name: aws-provider
+    namespace: ${INFRA_NAMESPACE}
+EOF
+
+# create IAM Role Policy objects in Crossplane, and wait until the corresponding
+# IAM Role Policies in AWS are created and Ready to use
+apply_and_wait_until_ready "${WORK_DIR}/policies.yaml"
+```
+
+<details> <summary>output</summary>
+
+```bash
+iamrolepolicyattachment.identity.aws.crossplane.io/aws-network-config-role-servicepolicy created
+iamrolepolicyattachment.identity.aws.crossplane.io/aws-network-config-role-clusterpolicy created
+iamrolepolicyattachment.identity.aws.crossplane.io/aws-network-config-role-servicepolicy condition met
+iamrolepolicyattachment.identity.aws.crossplane.io/aws-network-config-role-clusterpolicy condition met
+```
+
+</details>
+
+<blockquote></details>
+
+## Configure Provider Resources
+
+Once we have the network configuration set up, we also need to tell Crossplane
+how to satisfy WordPress's claims for a database and a Kubernetes cluster, using
+AWS resources. The following resource classes allow the EKSCluster and
+RDSInstance claims to be satisfied, using the network configuration we created
+in the previous step:
+
+```bash
+# build resource classes yaml, by using the configured network resources
+cat > ${WORK_DIR}/resource_classes.yaml <<EOF
+---
+apiVersion: database.aws.crossplane.io/v1alpha2
+kind: RDSInstanceClass
+metadata:
+  name: standard-mysql
+  namespace: ${INFRA_NAMESPACE}
+specTemplate:
+  class: db.t2.small
+  masterUsername: masteruser
+  securityGroups:
+    - ${RDS_SECURITY_GROUP_ID}
+  subnetGroupName: ${RDS_SUBNET_GROUP_NAME}
+  size: 20
+  engine: mysql
+  providerRef:
+    name: aws-provider
+    namespace: ${INFRA_NAMESPACE}
+  reclaimPolicy: Delete
+---
+apiVersion: compute.aws.crossplane.io/v1alpha2
+kind: EKSClusterClass
+metadata:
+  name: standard-cluster
+  namespace: ${INFRA_NAMESPACE}
+specTemplate:
+  region: ${AWS_REGION}
+  roleARN: ${EKS_ROLE_ARN}
+  vpcId: ${VPC_ID}
+  subnetIds:
+    - ${SUBNET1_ID}
+    - ${SUBNET2_ID}
+    - ${SUBNET3_ID}
+  securityGroupIds:
+    - ${CLUSTER_SECURITY_GROUP_ID}
+  workerNodes:
+    nodeInstanceType: m3.medium
+    nodeAutoScalingGroupMinSize: 1
+    nodeAutoScalingGroupMaxSize: 1
+    nodeGroupName: demo-nodes
+    clusterControlPlaneSecurityGroup: ${CLUSTER_SECURITY_GROUP_ID}
+  providerRef:
+    name: aws-provider
+    namespace: ${INFRA_NAMESPACE}
+  reclaimPolicy: Delete
+EOF
+
+# apply the resource classes yaml to Crossplane
+kubectl apply -f "${WORK_DIR}/resource_classes.yaml"
+```
+
+So far we have been creating resources in `INFRA_NAMESPACE`, where all resources
+are to configure AWS stack with the AWS account, create a network configuration,
+and define resources classes that will satisfied the claims. Now, we will create
+an app namespace and populate it with resources that are used to let Crossplane
+know how to satisfy the claims. Let's call this namespace `myapp`
```

Comment:
We'll want to change this namespace to be more consistent with the other sections of the guide; they are using `app-project1-dev`